ibm_catalog.json

```diff
@@ -107,20 +107,12 @@
                 },
                 {
                   "key": "plan",
-                  "default_value": "free",
+                  "default_value": "standard",
```

Comment:
dont add a default value here, otherwise we need to maintain it in terraform and here. By leaving it out it defaulss to terraform default

Reply:
done